CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
+
+The project currently does not use Conventional Commits or any specific conventions.
+Work is branched
+and merged from [main](https://github.com/algorandfoundation/devportal/tree/devrel-content) as the main trunk.
+
+For the internal devrel guide,
+see the [Solas documentation](https://solas.algorand.foundation/doc/content-development-workflow-dPylBlDwdm#h-feature-branches)
+
+## Get Started
+
+### Setup Repository
+
+Clone the repository if it has not been done already.
+This only has to be done once to set up the project
+
+```bash
+git clone git@github.com:algorandfoundation/devportal-code-examples.git
+```
+
+Update the latest changes from the remote repository
+
+```bash
+git fetch
+```
```

Comment:
We could go further here and bring the developer all the way up with the project.

I would add:

- Starting localnet (`algokit localnet start`)
- Bootstrapping all of the projects for the first time (`algokit project bootstrap all`)
- Running the tests (`algokit project run test`)